.goreleaser.yaml

```diff
@@ -68,14 +50,19 @@ builds:
       -X github.com/anchore/syft/internal/version.gitTreeState={{.Env.BUILD_GIT_TREE_STATE}}
 
 archives:
-  - format: tar.gz
-    builds:
-      - syft # i.e. Linux only
+  - name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    id: syft
+    replacements:
+      windows: Windows
+      amd64: x86_64
```

Comment:
Hi guys, evidently this has messed up artifact names for Linux and MacOS on amd64 architecture.

Release [v0.35.1](https://github.com/anchore/syft/releases/tag/v0.35.1):
```
syft_0.35.1_linux_amd64.tar.gz
```

Release [v0.36.0](https://github.com/anchore/syft/releases/tag/v0.36.0):
```
syft_0.36.0_darwin_x86_64.tar.gz   ## this one was not there before and is likely redundant
syft_0.36.0_linux_x86_64.tar.gz
```